packages/suite/src/actions/firmware/firmwareActions.ts

```diff
@@ -111,29 +110,40 @@ const firmwareInstall =
                 console.warn(`Installing firmware ${toRelease.release.version}`);
```

Comment:
e1fe2661babdd65547300324d2557ddd0ac053fb

Reply:
Probably different commit. I miss `toRelease.release.version.join('.')` the `join` part

Reply:
Sorry, it's here: fbaa7b408709cc40ccc07d507897ce22d7a57840

Reply:
It is not.

```
            console.warn(
                intermediary
                    ? 'Cannot install latest firmware. Will install intermediary fw instead.'
                    : `Installing ${
                          toBitcoinOnlyFirmware ? FirmwareType.BitcoinOnly : FirmwareType.Universal
                      } firmware ${toRelease.release.version}.`,
            );
```
It misses the `join` part, so there are `,` instead of `.` in console

Reply:
Ok I see: bd6ecce2e760747026e041fbc6414e47ee7d5519